packages/discord.js/src/structures/ChatInputCommandInteractionOptionResolver.js

```diff
@@ -1,21 +1,16 @@
 'use strict';
 
 const { ApplicationCommandOptionType } = require('discord-api-types/v10');
-const { TypeError, ErrorCodes } = require('../errors');
+const InteractionOptionResolver = require('./InteractionOptionResolver');
+const { ErrorCodes } = require('../errors');
```

Comment:
You should still import the TypeError here!

Reply:
Hi